### ISSUE TITLE
m1 mac gets javascript error on load

### ISSUE THREAD

**Issue by arthyn** [jalehman/port#207]
```
Error: node-loader:
Error: dlopen(/Applications/port.app/Contents/Resources/app/.webpack/main/e961c6402c35d4a4236342d60ab89149.node, 0x0001): tried: '/Applications/port.app/Contents/Resources/app/.webpack/main/e961c6402c35d4a4236342d60ab89149.node' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e'))
at Object.879 (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:100658)
at i (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:296567)
at Object.2807 (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:105509)
at i (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:296567)
at Object.8892 (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:101436)
at i (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:296567)
at Object.769 (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:142493)
at i (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:296567)
at Object.9979 (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:128052)
at i (/Applications/port.app/Contents/Resources/app/.webpack/main/index.js:2:296567)```
```

**Comment by tastem-pacper** [comment]
Edit: Reproduced the error I mentioned here and it doesn't seem to be related to the above so I've created a new issue: #208

**Comment by arthyn** [comment]
@tastem-pacper what was the error?

**Comment by tastem-pacper** [comment]
Trying to repro, will update. Edit: #208

**Comment by aaronfc** [comment]
Happening also to me too when using the `Port-arm64.dmg` from [v1.9.0 release](https://github.com/urbit/port/releases/tag/v1.9.0).

**Comment by latter-bolden** [comment]
Thanks for reporting this!

 Looks like one of the dependencies isn't getting compiled correctly. We had a previous iteration of the Apple Silicon build working, so we're looking into what might have changed.

Will try to have this resolved for the next patch release. In the meantime, I'd recommend using the default non-M1 build through [Rosetta](https://support.apple.com/en-gb/HT211861).

**Comment by aaronfc** [comment]
>Will try to have this resolved for the next patch release. In the meantime, I'd recommend using the default non-M1 build through [Rosetta](https://support.apple.com/en-gb/HT211861).

To be honest I tried installing the arm64 from the github release page because I couldn't get the `port.dmg` [from  the homepage](https://urbit.org/getting-started/desktop) to work. I got a black screen after creating a comet (launchpad).

I will wait for the new patch version to try again :D, thanks for all your effort @latterbolden

**Comment by latter-bolden** [comment]
@aaronfc happy to help! How long were you stuck at the black screen?

If you were able to boot the comet successfully (i.e. "Launch Landscape" button popped up or green status from the ship list), it should be working. The black screen usually occurs when Urbit is clogged and can't respond instantaneously to the web request. This sometimes happens after the first boot when the ship is handling all of it's initial updates, but shouldn't be a persistent issue.

**Comment by aaronfc** [comment]
I waited for more than 15 minutes. But I will give it a go again after reinstalling.

Edit: Just tested and even though I had some problems starting a comet I finally got to the Launchpad correctly 😄

**Comment by latter-bolden** [comment]
Glad to hear you were able to get up and running!

15 minutes is unusual and we've had a few other reports of longer than expected blank screens, so I'm going to investigate.

**Comment by kingdonb** [comment]
🎉 that this can be closed, however

The latest release of port is still from ~25 days ago

I was able to get my comet going using port and the dev instructions from the README, nodejs 16, yarn install, etc. plus finally "npm run start" – I am not sure if I'm using Rosetta or not, but I enabled it, and everything seems to work now on my M1 mac. The distro of port that I downloaded from the Releases errors out and will not load. I was able to get what I needed from the comet, all good on my end now.

Hope everything is good and looking forward to a next release soon 👍
